src/components/CheckboxFieldItem/CheckboxFieldItem.tsx

```diff
@@ -112,25 +113,21 @@ export const CheckboxFieldItem = ({
     className,
     inverted && styles['checkbox-field__item--inverted'],
   );
+  const label = (
```

Comment:
could we just pass `text` and remove the styling? Styling should happen through the `Checkbox` component now right

Reply:
Definitely could, and will change to do so, was just being respectful of the custom styling that was here